docs/guides/high-availability/concepts-threads.adoc

```diff
@@ -21,6 +21,15 @@ JGroups communications, which is used in single-site setups for the communicatio
 This reduces the memory usage and removes the need to configure thread pool sizes.
 Therefore, the use of OpenJDK 21 is recommended.
 
+=== JGroups communications
```

Comment:
this seems duplicated from line 15.

Reply:
Thank you for spotting this one, fixed. Please re-review.